fmp_data/helpers.py

```
@@ -36,3 +38,43 @@ def wrapped(*args: Any, **kwargs: Any) -> Any:
+    Returns:
+        A decorator that raises RemovedEndpointError when the function is called.
+
+    Example:
+        >>> @removed("This endpoint was discontinued by FMP in 2024.")
+        ... def old_method():
+        ...     pass
+    """
+
+    def decorator(func: F) -> F:
+        @functools.wraps(func)
+        def wrapped(*args: Any, **kwargs: Any) -> Any:
```

Ruff (0.14.13)

Unused function argument: `args`

(ARG001)